jobs/jobs.go

```diff
@@ -35,28 +38,63 @@ type JobStatus struct {
 }
 
 type Job struct {
-	Id                   string     `json:"id"`
```

Comment:
This PR is easiest reviewed with [whitespace hidden](https://github.com/AdRoll/batchiepatchie/pull/75/files?diff=unified&w=1).